integrations/docker/images/chip-cert-bins/README.md

```diff
@@ -0,0 +1,46 @@
+# Docker image for Matter Certification Test Harness
+
+The Dockerfile here helps build multi-platform Docker images containing the  executable binaries necessary for the Matter Test Harness. It utilizes the BuildKit toolkit and Buildx included within Docker since version 18.06.
+
+## Running
+
+In order to properly run the binaries, avahi must be properly set up and passed to the container.
+
+Prerequisites:
+
+* Host must support and enable IPv6 and be on a network that has IPv6.
+* IPv6 must be enabled within avahi config on the host. ```use-ipv6=yes``` in avahi-daemon.conf
+* Sometimes there are stale avahi entries, so restarting avahi-daemon between runs may be necessary.
+
+The host network and dbus must be exposed to the container for avahi to work properly. So for an interactive prompt, use:
+
+```docker run -it --network host -v /var/run/dbus/system_bus_socket:/var/run/dbus/system_bus_socket [image name or hash]```
```

Comment:
I think github requires a newline after triple backtick.

So here use spacing. Use single backtick when having inline code (like lines 26 and 27 and such)